src/app/helptext/apps/apps.ts

```diff
@@ -23,10 +23,17 @@ export default {
     loading: T('Loading...'),
     not_configured: T('Applications not configured'),
     not_running: T('Applications are not running'),
+    no_search_result: T('No Search Results.'),
     no_installed: T('No Applications Installed'),
     no_installed_message: T('Applications you install will automatically appear here. Click below and browse the TrueNAS catalog to get started.'),
   },
 
+  catalogMessage: {
+    loading: T('Loading...'),
+    no_search_result: T('No Search Results.'),
+    no_catalog: T('It was unable to retrieve catalogs data.'),
```

Comment:
Can you please clarify what `It` is for the user ?

Reply:
Just copied/pasted from your description. If you have any better one, just let me know.

Reply:
`UI is unable to retrieve Available Application(s)` - @truenas/docs-team can you please advise ?

This is for a case where for some reason we are unable to retrieve any available application ( that reason being erroring out very likely on the catalog(s) end ).